README.md

```diff
@@ -81,142 +81,13 @@ Note that kpod and its container management and debugging commands have moved to
-  containers-common \
-  git \
-  golang-go \
-  libassuan-dev \
-  libdevmapper-dev \
-  libglib2.0-dev \
-  libc6-dev \
-  libgpgme11-dev \
-  libgpg-error-dev \
-  libseccomp-dev \
-  libsystemd-dev \
-  libselinux1-dev \
-  pkg-config \
-  go-md2man \
-  runc
-```
-
-Debian, Ubuntu, and related distributions will also need a copy of the development libraries for `ostree`, either in the form of the `libostree-dev` package from the [flatpak](https://launchpad.net/~alexlarsson/+archive/ubuntu/flatpak) PPA, or built [from source](https://github.com/ostreedev/ostree) (more on that [here](https://ostree.readthedocs.io/en/latest/#building)).
-
-If using an older release or a long-term support release, be careful to double-check that the version of `runc` is new enough (running `runc --version` should produce `spec: 1.0.0`), or else build your own.
-
-**NOTE**
-
-Be careful to double-check that the version of golang is new enough, version 1.8.x or higher is required.  If needed, golang kits are available at https://golang.org/dl/
-
-### Get Source Code
-
-Clone the source code using:
-
-```bash
-git clone https://github.com/cri-o/cri-o # or your fork
-cd cri-o
-```
-
-### Build
-
-```bash
-make install.tools
-make
-sudo make install
-```
-
-Otherwise, if you do not want to build `CRI-O` with seccomp support you can add `BUILDTAGS=""` when running make.
-
-```bash
-make install.tools
-make BUILDTAGS=""
-sudo make install
-```
-
-#### Build Tags
-
-`CRI-O` supports optional build tags for compiling support of various features.
-To add build tags to the make option the `BUILDTAGS` variable must be set.
-
-```bash
-make BUILDTAGS='seccomp apparmor'
-```
-
-| Build Tag | Feature                            | Dependency  |
-|-----------|------------------------------------|-------------|
-| seccomp   | syscall filtering                  | libseccomp  |
-| selinux   | selinux process and mount labeling | libselinux  |
-| apparmor  | apparmor profile support           | <none>      |
-
-### Running pods and containers
-
-Follow this [tutorial](tutorial.md) to get started with CRI-O.
-
-### Setup CNI networking
-
-A proper description of setting up CNI networking is given in the
-[`contrib/cni` README](contrib/cni/README.md). But the gist is that you need to
-have some basic network configurations enabled and CNI plugins installed on
-your system.
-
-### Running with kubernetes
-
-You can run a local version of kubernetes with CRI-O using `local-up-cluster.sh`:
-
-1. Clone the [kubernetes repository](https://github.com/kubernetes/kubernetes)
-1. Start the CRI-O daemon (`crio`)
-1. From the kubernetes project directory, run:
-```shell
-CGROUP_DRIVER=systemd \
-CONTAINER_RUNTIME=remote \
-CONTAINER_RUNTIME_ENDPOINT='unix:///var/run/crio/crio.sock  --runtime-request-timeout=15m' \
-./hack/local-up-cluster.sh
-```
-
-To run a full cluster, see [the instructions](kubernetes.md).
+[If you're eager to get started, check out our setup guide here](tutorials/setup.md)
```

Comment:
To build crio from source, check out our setup guide here.

Reply:
Also, we should retain the kubernetes local-up-cluster on the main page itself for a quick start experience where one has the instructions to install from packages and then start a local cluster.